src/crimson/os/seastore/transaction_manager.cc

```diff
@@ -272,7 +276,8 @@ TransactionManager::submit_transaction_direct(
 
     DEBUGT("about to submit to journal", tref);
 
-    return journal->submit_record(std::move(record), tref.get_handle()
+    return jm->submit_record(std::move(record), tref.get_handle(),
+      segment_manager.get_device_id()
```

Comment:
IIUC, this pr only uses SegmentedJournal, CBJournal hasn't been involved in journaling, right?